coursier.bzl

```diff
@@ -346,7 +367,29 @@ def _pinned_coursier_fetch_impl(repository_ctx):
 
     dep_tree = maven_install_json_content["dependency_tree"]
 
-    dep_tree_signature = dep_tree.get("__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY")
+    # Then, check to see if we need to repin our deps because inputs have changed
+    if dep_tree.get("__INPUT_ARTIFACTS_HASH") == None:
+        print("NOTE: %s_install.json does not contain a signature of the required artifacts. " % repository_ctx.name +
+              "This feature ensures that the build does not use stale dependencies when the inputs " +
+              "have changed. To generate this signature, run 'bazel run @unpinned_%s//:pin'." % repository_ctx.name)
+    else:
+        computed = compute_dependency_inputs_signature(repository_ctx.attr.artifacts)
```

Comment:
nit: mind calling this `computed_artifacts_hash` ?

Reply:
Done